### DISCUSSION TITLE
How to do static Github Pages deployment with Nuxt3?

### DISCUSSION THREAD

**Discussion by pratyushtewari** [nuxt/nuxt#16157] [Questions]
Are there is any instructions to how to deploy the static website to github pages? There is [one on Nuxt2](https://nuxtjs.org/deployments/github-pages/) but I cannot figure out how to do this with Nuxt3.

**Comment by sikhlana** [comment]
Nuxt3's static site generation is still under development as far as I know.

**Comment by kutaycoskuner** [comment]
Any progression so far?

**Comment by kuzroman** [comment]
indeed need instructioin!

**Comment by malgamves** [comment]
If I go by the docs, looks like you have to set `ssr: false` in your `nuxt.config.ts` and then run `nuxi generate` to generate your static files.
